docs/textual_definitions_SOP.md

```diff
@@ -0,0 +1,45 @@
+## Guide to writing textual definitions on CL
+
+Links to relevant background material:
+
+- Chris Mungall's [blog on post ontology definitions](https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/)
+- [OBO foundry reference paper](https://philpapers.org/archive/SEPGFW.pdf);
+- DOS slides on the problem of defining cell types (TBA)
+
+### Background
+
+It is standard ontology engineering practise to aim for minimal, concise ontology term definitions.  However many cell types can be reliably identified by more than one set of properties: functional, structural, gene expression. This makes it hard to choose which properties to include if we are aiming for a minimal definition.  Users of the cell ontology also come from different disciplines/perspectives and have different types of information and levels of detail available when they annotate a term or browse a resource.  We need to be able to support users from multiple disciplines with a definition that allows them to visualise and identify the cell type being defined. In some cases (e.g. transcriptomically defined types or 't-types') identification of the cell type  require links to reference data.
+
+Here are a couple of examples of minimal definitions that are correct but not useful to most users:
+
+1. We could minimally define a corneal endothelial cell as 'Any endothelial cell that is part of the cornea'.  This may well be sufficient for an expert in the anatomy and biology of the cornea, but to most biologists, the term "endothelial cell" brings to mind a the principle cell types of lymphatic or blood vessels.  However, the corneal endothelium is a monolayer of flat cell on the underside of the cornea.    
+2. Similarly, a prefectly accurate minimal definition of a type II pneuomocyte is an epithelial cell that has an 'alveolar lamellar body' (a unique structure only found in these cell types). But this is useless information to a use who knows nothing about this structure (probably most biologists) or who has to annotated data that does not resolve this structure.
+
+A second use of ontologies is encode to knowledge in the form of useful formal links between ontology terms.  For example, in CL we record function and cell components via links to gene ontology terms, location via links to CL and lineage via links to other CL terms. Not all of this information is particularly useful for recognising a cell type, but it is of use to our users and so we often record it in CL using formal relationships. This is relevant to ontology definitions because it is good practise for formal and textual definitions to match, and textual definitions are the place we encode supporting references.
```

Comment:
```suggestion
A second use of ontologies is to encode knowledge in the form of useful formal links between ontology terms.  For example, in CL we record function and cell components via links to gene ontology terms, location via links to CL and lineage via links to other CL terms. Not all of this information is particularly useful for recognising a cell type, but it is of use to our users and so we often record it in CL using formal relationships. This is relevant to ontology definitions because it is good practise for formal and textual definitions to match, and textual definitions are the place we encode supporting references.
```